docs/docsite/rst/roadmap/COLLECTIONS_9.rst

```diff
@@ -40,7 +40,7 @@ Release schedule
 
 .. [2] After this date only changes blocking a release are accepted. Accepted changes require creating a new release candidate and may slip the final release date.
 
-.. [3] Collections will be updated to a new version only if a blocker is approved. Collection owners should discuss any blockers at a community IRC meeting (before this freeze) to decide whether to bump the version of the collection for a fix. See the `Community IRC meeting agenda <https://github.com/ansible/community/issues/539>`_.
+.. [3] Collections will be updated to a new version only if a blocker is approved. Collection owners should discuss any blockers with a ``community_wg`` forum topic(before this freeze) to decide whether to bump the version of the collection for a fix. See the `Creating a Community working group topic <https://github.com/ansible-community/community-topics/blob/main/community_topics_workflow.md#creating-a-topic>`_ for details.
```

Comment:
```suggestion
.. [3] Collections will be updated to a new version only if a blocker is approved. Collection owners should discuss any blockers with a ``community-wg`` tagged forum topic (before this freeze) to decide whether to bump the version of the collection for a fix. See the `Creating a Community working group topic <https://github.com/ansible-community/community-topics/blob/main/community_topics_workflow.md#creating-a-topic>`_ for details.
```

Reply:
I don't understand why we should have a change here at all. I looks to be specifically for RC1 (or maybe any other RCs?) but since 9 has already been released I would keep it as-is.

Reply:
it likely didn't need to be updated,  but wanted it 'clean' in regards to information on forum vs the old community topics.

Reply:
I wouldn't change this. We didn't discuss those things in the forum because (it didn't exist yet), did we?

Anyway, if you want to change this for Ansible 9 because you want to have it "'clean' in regards to information on forum vs the old community topics", it should also be changed for the roadmaps < 9. And this makes even less sense to me...

Reply:
@felixfontein can you respond here as it was your suggestion to change the Ansible 9 roadmap to point to the forum?

Reply:
Hmm, @mariolenz has a good point (the time that that footnote was relevant is over).

On the other hand, the older roadmaps are for EOL versions of Ansible, while the Ansible 9 roadmap is for the currently latest version (and if https://forum.ansible.com/t/vote-closes-on-2023-01-29-maintain-ansible-9-for-longer-than-6-months/2574/17 passes, it will get more updates for half a year longer than initially planned).

So I do see a good reason to update the Ansible 9 roadmap as well, since users might look at it and want to start a discussion on some aspect of Ansible 9, and see that note (which is meant for a specific situation that is already over, but they might simply ignore that) and end up creating an issue in the repo instead of a forum post.